### PR TITLE
Adjust KVM container instructions

### PR DESCRIPTION
**Pull request by Fuzzy-Math**
### Description

Doc changes in preparation of the split into two separate kvm containers (kvm server and kvm client)

Add new instructions for kvm client container

Rename kvm-container-manage script references to kvm-server-manage

### Is this (based on) existing content?
https://susedoc.github.io/doc-modular/main/html/alp-dolomite/available-alp-workloads.html#task-run-kvm-with-podman

### CODE REVIEW

**Review by hw-claudio — APPROVED** [review]
lgtm

**Comment by tbazant** [comment]
Thanks a lot @Fuzzy-Math for the contribution. I made several changes to the PR, look at it please and let me know if they're OK, then i'll merge & publish it.

**Comment by Fuzzy-Math** [comment]
@tbazant, I've added a few more small edits. If everything looks fine, would you like me to squash these commits or will you squash during the PR merge?

**Comment by tbazant** [comment]
Adding @dariavladykina for language proofing

File: tasks/run-kvm-client-with-podman.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      <para>
+        Optionally, print the list of installed wrappers.
+      </para>
+<screen>&prompt.root;find -L /usr -xtype l -samefile $(command -v kvm-client-wrapper)</screen>
+    </step>
+    <step>
+      <para>
+        Run the desired tool from the list of installed wrappers. The wrapper
+        transparently handles passing through the given executable, flags, and
+        arguments into the container
```
Comment (dariavladykina) [inline tasks/run-kvm-client-with-podman.xml:58]:
```suggestion
        arguments into the container.
```

File: tasks/run-kvm-with-podman.xml [hunk 2]
```diff
@@ -37,88 +37,59 @@
...
+          Deploy the &kvmserver; workload.
         </para>
-<screen>&prompt.root;kvm-container-manage.sh create</screen>
-      </step>
-      <step>
+<screen>&prompt.root;kvm-server-manage enable</screen>
         <para>
-          Start the container:
+          Refer to <xref linkend="reference-kvm-server-manage"/> for more info
+          on managing the deployment of the &kvmserver; workload
```
Comment (dariavladykina) [inline tasks/run-kvm-with-podman.xml:72]:
```suggestion
          on managing the deployment of the &kvmserver; workload.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, all good, just a couple of full stops missing. Thanks!

**Pull request merged**